models/recorder/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@hcengineering/model-recorder",
+  "version": "0.6.0",
+  "main": "lib/index.js",
+  "svelte": "src/index.ts",
+  "types": "types/index.d.ts",
+  "author": "Hardcore Engineering Inc.",
+  "template": "@hcengineering/model-package",
+  "license": "EPL-2.0",
+  "scripts": {
+    "build": "compile",
+    "build:watch": "compile",
+    "format": "format src",
+    "_phase:build": "compile transpile src",
+    "_phase:format": "format src",
+    "_phase:validate": "compile validate"
+  },
+  "devDependencies": {
+    "@hcengineering/platform-rig": "^0.6.0",
+    "@typescript-eslint/eslint-plugin": "^6.11.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-n": "^15.4.0",
+    "eslint": "^8.54.0",
+    "@typescript-eslint/parser": "^6.11.0",
+    "eslint-config-standard-with-typescript": "^40.0.0",
+    "prettier": "^3.1.0",
+    "typescript": "^5.3.3"
+  },
+  "dependencies": {
+    "@hcengineering/core": "^0.6.32",
+    "@hcengineering/model": "^0.6.11",
+    "@hcengineering/model-core": "^0.6.0",
+    "@hcengineering/model-presentation": "^0.6.0",
+    "@hcengineering/model-workbench": "^0.6.1",
```

Comment:
Could you check if all dependencies are required?
For example, I don't see usage of model-presentation and model-workbench

Reply:
thanks, fixed